client/src/elements/item-preview/item-preview.js

```diff
@@ -64,6 +53,14 @@ export class ItemPreview {
 
   async init() {
     try {
+      // set the preview size options
+      const previewSizes = await this.qConfig.get("previewSizes");
+      this.sizeOptions = previewSizes.map(previewSize => {
+        return {
+          value: previewSize.value,
+          text: this.i18n.tr(previewSize.translationKey)
```

Comment:
do not translate the string here but use the `t` binding behaviour in the template. see https://aurelia.io/docs/plugins/i18n#using-the-plugin `Translating with the TBindingBehavior` for details.

With this binding behaviour the string updates when the language is switched.

Reply:
Implemented in https://github.com/nzzdev/Q-editor/pull/119/commits/a3daeac9893dccf9d03a2830bf0bd9c4de25a04c